test/GitcoinGovernor.t.sol

```diff
@@ -29,3 +36,216 @@ contract GitcoinGovernorTest is Test, DeployInput {
     assertEq(governor.COUNTING_MODE(), "support=bravo&quorum=for,abstain");
   }
 }
+
+contract GitcoinGovernorProposalTest is GitcoinGovernorTest {
+
+  IGovernorAlpha governorAlpha = IGovernorAlpha(0xDbD27635A534A3d3169Ef0498beB56Fb9c937489);
```

Comment:
Switched it to a public constant and tried reading it from the propose script after deployment instead and the tests now revert 🤔 

```
2022-12-02T17:53:58.545091Z ERROR forge::runner: setUp failed reason="EvmError: Revert" contract=0x7fa9385be102ac3eac297483dd6233d62b3e1496
```

Not clear why but not sure it's worth spending time debugging atm since they match and won't ever change.